db/permissions_roles.yml

```diff
@@ -39,42 +33,29 @@ Reviewer:
 
 Supervisor:
 - can_manage_packages
-- can_manage_package_detail
 - can_manage_offers
-- can_manage_goodcity_requests
 - can_manage_deliveries
-- can_manage_orders
```

Comment:
For all these permissions that are removed from Supervisor and Reviewer, we need to make sure they are added to Stock fulfillment / Stock Administrator above.